packages/webpack/src/plugin.ts

```diff
@@ -0,0 +1,100 @@
+//@angular/webpack plugin main
+import {ReflectiveInjector, OpaqueToken} from '@angular/core'
+import * as ts from 'typescript'
+import * as ngCompiler from '@angular/compiler-cli'
+import * as tscWrapped from '@angular/tsc-wrapped'
+import * as webpack from 'webpack'
+import * as path from 'path'
+import * as fs from 'fs'
+import * as MemoryFileSystem from 'memory-fs'
+import {WebpackResourceLoader} from './resource_loader'
+import {createCodeGenerator} from './codegen'
+import * as astTools from '../../../dist/ast-tools/index'
```

Comment:
Could you use `@angular-cli/ast-tools` and add the path mapping?  This would break when published.